docs/static/_redirects

```diff
@@ -3,3 +3,22 @@
 # The test-utils lived at its current home for about 5 days.
 # It at least exists on some slides and Tweets.
 /docs/apollo-server/features/test-utils.html /docs/apollo-server/features/testing.html
+
+# Redirects associated with revamp of Apollo Server information architecture
```

Comment:
Quite glad you've taken the time to set these up.  Super important for sure, though I know they can be frustrating to sort out. 😄